php/libraries/OutputWrapper.class.inc

```diff
@@ -0,0 +1,190 @@
+<?php declare(strict_types=1);
+require_once 'generic_includes.php';
+/**
+ * This class is designed to assist in writing command-line interface tools
+ * for LORIS.
+ *
+ * It contains helper functions to format output, etc.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class OutputWrapper
+{
+    // Colours associated with the bash shell.
+    // @see https://misc.flogisoft.com/bash/tip_colors_and_formatting
+    const COLOURS = [
+        'red'    => '31',
+        'yellow' => '33',
+        'green'  => '32',
+    ];
+    /**
+     * Whether this script should log its output to a file or just print to
+     * STDOUT.
+     *
+     * @var bool
+     */
+    private $_loggingEnabled = false;
+    /**
+     * The directory that will contain log files written by this script.
+     *
+     * @var SPLFileInfo
+     */
+    private $_logDirectory;
+
+    /**
+     * Create a new command line helper.
+     *
+     * @param ?SPLFileInfo $logDirectory The directory that will contain log
+     *                                   files written by this script.
+     *
+     * @return void
+     */
+    public function __construct(SPLFileInfo $logDirectory = null)
+    {
+        $this->_logDirectory = new SPLFileInfo(
+            $logDirectory ?? \NDB_Factory::singleton()->config()->getSetting('log')
```

Comment:
This should probably check that the resulting directory is writeable and is a directory.

Reply:
We should probably also modify the Log config setting in the SQL table to be a `web_path` since that will allow the Configuration module to validate this when changed on the front-end.